### PR TITLE
Desktop 4.17.1 release notes

### PR DESCRIPTION
**Pull request by mat007**
### Proposed changes

Added release notes for the Windows only release of Docker Desktop 4.17.1

### CODE REVIEW

File: desktop/release-notes.md [hunk 1]
```diff
@@ -24,13 +24,78 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+            <i class="fa fa-chevron-down"></i>
+          </a>
+        </h5>
+      </div>
+      <div id="collapse16" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading16">
+        <div class="panel-body">
+          <a class="btn btn-primary" href="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" role="button">Download file</a> 
+        <br>
+        <br>
+        <b>Checksum:</b>
```
Comment (mat007) [inline desktop/release-notes.md:48]: We are still waiting for the checksum here.

File: desktop/release-notes.md [hunk 2]
```diff
@@ -24,13 +24,78 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+> For the latest release of Docker Desktop for Mac and Linux, see [4.17.0](#4170)
+ 
+### Bug fixes and enhancements
+
+#### For Windows
+
+- Allowed Windows containers to work when BitLocker is enabled on C:
+- Fixed a bug where `docker buildx` container builders would lose access to the network after 24hrs.
+- Fixed a bug where Registry Access Management policy updates were not downloaded.
+
```
Comment (djs55) [inline desktop/release-notes.md:63]:
```suggestion
- Improved debug information to better characterise failures under WSL 2.
```
(based on https://docker.slack.com/archives/C04QCKUKLBH/p1678109131361909?thread_ts=1678105027.819099&cid=C04QCKUKLBH)

File: desktop/release-notes.md [hunk 3]
```diff
@@ -24,13 +24,78 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+    </div>
+    </div>
+>
+> For the latest release of Docker Desktop for Mac and Linux, see [4.17.0](#4170)
+ 
+### Bug fixes and enhancements
+
+#### For Windows
+
+- Allowed Windows containers to work when BitLocker is enabled on C:
```
Comment (aevesdocker) [inline desktop/release-notes.md:60]:
```suggestion
- Docker Desktop now allows Windows containers to work when BitLocker is enabled on C:
```

File: desktop/release-notes.md [hunk 4]
```diff
@@ -24,13 +24,78 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.17.1
+
+{% include release-date.html date="2023-02-27" %}
```
[the remark this answers is not in the document]
Comment (mat007) [reply desktop/release-notes.md:29]: Oh it’s actually going to be published on Monday. I’ll update the date, thanks!

**Review by aevesdocker — APPROVED** [review]

**Review by dgageot — APPROVED** [review]

**Pull request merged**